src/main/java/com/org/checkr/service/JwtService.java

```diff
@@ -67,6 +65,8 @@ private String createToken(Map<String, Object> claims, String userName) {
 
 
     private Key getSignKey() {
+        Map<String, String> envVars = System.getenv();
```

Comment:
can u access env values using configuration like @Value instead of using getEnv

Reply:
done